### PR TITLE
Add integration tests to Visualizer

### PR DESCRIPTION
**Pull request by matheuscmelo**
This PR is able to add integration tests to Visualizer's CI.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,10 +1,29 @@
 pipeline {
   agent any
   stages {
-    stage('') {
+    stage('Unit') {
+      agent any
       steps {
-        sh 'tox -r'
```
Comment (rafaelvfalc) [inline Jenkinsfile:6]: Tox should not recreate the virtual environment in every build?

**Review by rafaelvfalc — APPROVED** [review]
Looks good, I just added I minor question about tox not recreate the virtual environment. However this is not a block for me. Thank you!

File: Jenkinsfile (hunk 1 above)
Comment (matheuscmelo) [reply Jenkinsfile:6]: Since Jenkins create a new workspace for each build, it doesn't need to recreate the env because there's no env in the first place

**Review by liliarsampaio — APPROVED** [review]
thanks, lgtm.

**Pull request merged**